src/main/java/com/icthh/xm/uaa/web/rest/AccountResource.java

```diff
@@ -77,6 +82,7 @@
     private final XmRequestContextHolder xmRequestContextHolder;
     private final TenantContextHolder tenantContextHolder;
     private final TenantPermissionService tenantPermissionService;
+    private final PasswordResetHandlerFactory resetFlowFactory;
```

Comment:
Maybe it worth moving PasswordResetHandlerFactory to UserService and simplify the Rest layer. 
@ishkurko @antonysychov what do you think?

Reply:
sure, good idea
done